docs/research/unraid-api-research.md

```
@@ -0,0 +1,819 @@
+
+### Enabling the GraphQL Sandbox
+
+Two methods:
+1. **WebGUI:** Settings > Management Access > Developer Options > Toggle GraphQL Sandbox to "On"
+2. **CLI:** `unraid-api developer --sandbox true`
+
+Then access at `http://YOUR_SERVER_IP/graphql` to explore the schema via Apollo Sandbox.
+
+### Query Types
+
+#### System Information (`info`)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)